ext/http/00_serve.ts

```diff
@@ -712,11 +697,12 @@ function serve(arg1, arg2) {
   const wantsHttps = hasTlsKeyPairOptions(options);
   const wantsUnix = ObjectHasOwn(options, "path");
   const signal = options.signal;
-  const onError = options.onError ?? function (error) {
-    // deno-lint-ignore no-console
-    console.error(error);
-    return internalServerError();
-  };
+  const onError = options.onError ??
+    function (error) {
+      // deno-lint-ignore no-console
+      console.error(error);
+      return internalServerError();
+    };
```

Comment:
Lots of unrelated changes due to moving code around :D

Reply:
Sorry